Lab 8.ipynb

```diff
@@ -0,0 +1,652 @@
+      "text/plain": [
+       "<Figure size 1080x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "n_higgs = (100 / (100 + 20000)) * (len(high_lumi['mass']) / len(higgs['mass']))\n",
+    "n_qcd = (20000 / (100 + 20000)) * (len(high_lumi['mass']) / len(qcd['mass']))\n",
+    "\n",
+    "fig, ax = plt.subplots(1, 1, figsize = (15, 10))\n",
+    "\n",
+    "qcd_weight = np.array([n_qcd] * len(qcd['mass']))\n",
+    "higgs_weight = np.array([n_higgs] * len(higgs['mass']))\n",
+    "\n",
+    "ax.hist([qcd['mass'], higgs['mass']], bins = 40, density = False, stacked = True, weights = [qcd_weight, higgs_weight], range = [124.28, 127.56], color = ['deepskyblue', 'darkblue'])\n",
+    "ax.hist(high_lumi['mass'], bins = 40, density = False, alpha = .8, color = \"pink\", range = [124.28, 127.56])\n",
+    "\n",
+    "ax.legend([\"QCD\", \"Higgs\", \"High Luminosity Experiment\"])\n",
+    "ax.set_title(\"High Luminosity Stacked Histogram With Mass Selection\")\n",
+    "ax.set_xlabel(\"Mass\")\n",
+    "ax.set_ylabel(\"Probability\")\n",
+    "\n",
+    "plt.show()"
```

Comment:
This is mass cut in the optimal range determined from earlier, right? Explain where you got the bin number and range. Also the y-axis still shouldn't be probability